internal/devconfig/config.go

```diff
@@ -73,11 +70,15 @@ func readFromFile(path string) (*Config, error) {
 	return loadBytes(b)
 }
 
-func LoadConfigFromURL(url string) (*Config, error) {
-	res, err := http.Get(url)
+func LoadConfigFromURL(ctx context.Context, url string) (*Config, error) {
+	req, err := http.NewRequestWithContext(ctx, http.MethodHead, url, nil)
```

Comment:
this should be `http.MethodGet`, right?

On line 84, we are trying to read `res.Body` which `HEAD` requests don't return

Reply:
oof, good catch. Terrible copy pasta on my part